modules/shared/components/atoms/ImageUpload/imageUpload.module.css

```diff
@@ -0,0 +1,17 @@
+.inputfile {
+  @apply opacity-0 w-full h-full absolute cursor-pointer;
+}
+.label {
+  @apply box-border bg-grey-shd7 rounded-md border-2 border-accent border-dashed 
+  flex flex-row justify-center items-center 
+  relative
+  w-auto
+  py-mx;
+}
```

Comment:
you need to change the component height "59.06px" which doesn't match the height in the design "56px", and also the border width from "2px" to "1px"